tox.ini

```diff
@@ -1,32 +1,45 @@
 [tox]
 envlist = py{34,35,36,37},docs,packaging
 skip_missing_interpreters = True
+minversion = 3.5.0
+isolated_build = true
 
-# NOTE: Don't use 'deps = .[<extra-requirements>]' tox option since we want Tox
-# to install the package from sdist first
+# Configure special Tox environment used for isolated building of the source
+# package.
+[testenv:.package]
+# Use the default Python 3 version available on the system.
+# NOTE: This makes it portable to CI systems where only one particular Python 3
+# version is installed.
+basepython = python3
 
 [testenv]
+basepython = python3
+extras =
+    docs: docs
+    !docs: test
+commands_pre =
+    # Verify installed packages have compatible dependencies.
+    # NOTE: Running 'pip check' after installation is necessary since pip
+    # currently lacks dependency resolution which means it can silently create
+    # broken installations.
+    # For more details, see: https://github.com/pypa/pip/issues/988.
+    pip check
+
 commands =
-# install testing requirements
-    pip install .[test]
-# run tests
-    py.test --verbose --cov=resolwe_runtime_utils
+    # General tests commands:
+    # Run tests.
+    py34: pytest --verbose --cov=resolwe_runtime_utils
```

Comment:
You can probably write this in a single line using `py3{4,5,6,7}`

Reply:
done. testing...